ucm2/Intel/sof-hda-dsp/dsp.conf

```diff
@@ -26,6 +26,12 @@ Define {
 	PostMixerAnalogPlaybackFirBytes "not available"
 	PostMixerAnalogPlaybackDrcBytes "not available"
```

Comment:
Nitpick: Commit message: either `adds` or `enable` in `This patch adds enable of the microphone` ?